<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="hp" failures="1" tests="6" name="Test" time="0.076" errors="0" timestamp="2023-06-21T14:38:36 IST">
  <testcase classname="testngdiscussion.TestngAllKeyword" name="LaunchBrowser" time="0.011"/>
  <testcase classname="testngdiscussion.TestngAllKeyword" name="LaunchBrowser" time="0.002"/>
  <testcase classname="testngdiscussion.TestngAllKeyword" name="Login" time="0.001">
    <failure type="java.lang.NullPointerException" message="login test case get failed">
      <![CDATA[java.lang.NullPointerException: login test case get failed
at testngdiscussion.TestngAllKeyword.Login(TestngAllKeyword.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="testngdiscussion.TestngAllKeyword" name="homepage" time="0.001"/>
  <testcase classname="testngdiscussion.TestngAllKeyword" name="orderpage" time="0.0">
    <skipped/>
  </testcase> <!-- orderpage -->
  <testcase classname="testngdiscussion.TestNgAllkeyword2" name="Logout" time="0.001"/>
</testsuite> <!-- Test -->
